src/azul/service/avro_pfb.py

```diff
@@ -261,8 +263,18 @@ def to_entity(cls, entity: PFBEntity) -> Self:
         return cls(dst_id=entity.id, dst_name=entity.name)
 
 
-def pfb_metadata_entity(entity_types: Iterable[str],
-                        links: bool = True
+def pfb_links_from_field_types(field_types: FieldTypes) -> MutableJSON:
```

Comment:
AFAICT, this is used twice and both time to compute an argument to pfb_metadata_entity, and only that. IOW, it could be inlined into the latter. The return type is too unspecific to add any clarity. I'm happy to elaborate in PL why this is an anti-pattern.

Reply:
Agreed at PL that this isn't an issue.